src/helpers/utils.ts

```diff
@@ -65,6 +66,19 @@ export function hasStrategyOverride(strategies: any[]) {
   return keywords.some(keyword => strategiesStr.includes(`"name":${keyword}`));
 }
 
+export function validateChoices({ type, choices }): boolean {
+  if (type && choices.length > 0) {
+    switch (type) {
+      case 'basic':
+        return isEqual(['For', 'Against', 'Abstain'], choices);
+      default:
+        return choices.length > 0;
```

Comment:
```suggestion
      default:
        return true;
```

Reply:
is it possible to have 0 choices?

Reply:
No, since it’s nested inside an if with choices > 0